iceoryx_posh/include/iceoryx_posh/internal/runtime/message_queue_interface.hpp

```diff
@@ -365,7 +367,7 @@ class MqRuntimeInterface
 
   private:
     std::string m_appName;
-    std::string m_segmentManager;
+    cxx::optional<RelativePointer::offset_t> m_segmentManagerAddressOffset;
```

Comment:
Very nice! I would have thought that in `m_segmentManager` some kind of name is stored but with this name it is now clear to me.